jbpm-test-coverage/src/test/filtered-resources/META-INF/persistence.xml

```diff
@@ -183,6 +198,76 @@
         <property name="hibernate.transaction.jta.platform" value="org.hibernate.service.jta.platform.internal.JBossStandAloneJtaPlatform" />
         <property name="hibernate.connection.handling_mode" value="DELAYED_ACQUISITION_AND_RELEASE_AFTER_TRANSACTION"/>
       </properties>        
-    </persistence-unit>    
+    </persistence-unit>
+
+  <persistence-unit name="org.jbpm.persistence.complete" transaction-type="JTA">
```

Comment:
Problem here is that `LogCleanupCommandTest` class needs a different PU for the AsyncJobListener.

https://github.com/kiegroup/jbpm/blob/master/jbpm-test-coverage/src/test/java/org/jbpm/test/functional/jobexec/LogCleanupCommandTest.java#L72

So if we move this `jbpm-test-util` we will still need to have a PU defined in the persistence.xml file specifically for this test case.

What do you think?

Reply:
Yes, the current solution is right. My point was that until now it worked just because `jbpm-executor` test-jar was on the classpath and the persistence.xml was taken from there. Moreover, this dependency is there just so the `CountDownAsyncJobListener` is available in the `jbpm-test-coverage` module. We can get rid of this dependency by putting that listener into the `jbpm-test-util` module where we currently have many other listeners. But this can be done in a separate PR later.

Reply:
sure. Will add that for a later PR